examples/with-apollo/lib/initApollo.js

```diff
@@ -1,27 +1,27 @@
-import { ApolloClient, createNetworkInterface } from 'react-apollo'
+import ApolloClient from 'apollo-client'
+import Link from 'apollo-link-http'
+import Cache from 'apollo-cache-inmemory'
 import fetch from 'isomorphic-fetch'
 
 let apolloClient = null
+let apolloCache = null
```

Comment:
variable is not used

Reply:
I'll be updating this example again once the work done https://github.com/apollographql/apollo-client/issues/2259 is available. Hopefully, that is either in the full release or the next beta update.

Reply:
@jthegedus Just checked that all the dependencies involving apollo client are released. So, maybe you can update the pull request and ask for merging too.

Reply:
@prateekrastogi Yep! I intend to update this within the next 24 hrs :smile: